spec/models/action_spec.rb

```diff
@@ -2,11 +2,81 @@
 
 describe Action do
   let(:page) { create :page }
+  let(:member) { create :member }
 
   describe 'on create' do
-    subject{ create(:action, page_id: page.id) }
+    describe 'member donor_status' do
+
+      describe 'when member is nondonor' do
+        before :each do
+          expect(member.donor_status).to eq 'nondonor'
```

Comment:
https://www.relishapp.com/rspec/rspec-core/v/2-2/docs/hooks/before-and-after-hooks

It's excessive, no? Isn't a single expectation describing the default state sufficient for you?